src/main/shell-sync.ts

```diff
@@ -25,7 +26,8 @@ export async function shellSync() {
   }
 
   if (!isSnap) {
-    process.env.PATH = env.PATH;
+    // Prefer the synced PATH over the initial one
+    process.env.PATH = unionPATHs(env.PATH ?? "",  process.env.PATH ?? "");
```

Comment:
What happens if both `env.PATH` and `process.env.PATH` are defined?  Or it's expected to be merged if so?

Reply:
That is expected, and yes it is expected that they will be merged.

Furthermore, the entries within each will be de-duplicated as well.